src/progress/progress.scss

```diff
@@ -5,7 +5,7 @@
   height: 8px;
   overflow: hidden;
   // stylelint-disable-next-line primer/colors
-  background-color: var(--color-auto-gray-2);
+  background-color: var(--color-border-tertiary);
```

Comment:
Before | After
--- | ---
![Screen Shot 2021-06-07 at 22 00 44](https://user-images.githubusercontent.com/378023/121023785-92883800-c7de-11eb-841c-599128f93905.png) | ![Screen Shot 2021-06-07 at 22 00 59](https://user-images.githubusercontent.com/378023/121023787-93b96500-c7de-11eb-8668-bc63ccd14fea.png)

It's not exactly the same for now. I initially used `var(--color-neutral-muted, var(--color-auto-gray-2))`, but `--color-neutral-muted` is not yet available and will be replaced at a later time.